src/core/server/models/user/helpers.ts

```diff
@@ -0,0 +1,11 @@
+import { STAFF_ROLES } from "coral-server/models/user/constants";
+
+import { User } from ".";
+
+export async function userIsStaff(user: User) {
```

Comment:
Function does not use `await`, therefore you can remove the `async` term. Also, scoping is helpful here:

```suggestion
export function userIsStaff(user: Pick<User, "role">) {
```

Reply:
Fixed this up, removed the async and also the await.